images/scripts/arch.setup

```diff
@@ -99,6 +99,10 @@ no-auto-default=*
 unmanaged-devices=interface-name:eth0
 EOL
 
+# Disable mDNS for systemd-resolved
```

Comment:
I think we should explain here why we switch it off (and in the commit message why it is needed now and not earlier).

Reply:
Added some explanation to the comment.